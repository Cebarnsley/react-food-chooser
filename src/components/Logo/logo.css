.logoContainer {
    display: flex;
}


.mainLogo {
    flex: 1;
    width: 80px;
    height: auto;
    margin: 0;
}
