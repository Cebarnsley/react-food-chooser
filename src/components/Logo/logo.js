import React from 'react';
import  './logo.css';

const Logo = () => {

 	return(
		<div className="logoContainer" >
			<img className="mainLogo" alt='food-chooser-logo' src={'/images/foodChooserLogo.png'} />
		</div>
		);
}

export default Logo;