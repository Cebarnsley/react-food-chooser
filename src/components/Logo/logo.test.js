import { shallow } from 'enzyme';
import React from 'react';
import Logo from './logo';

it('expect to render logo component', () => {
    expect(shallow(<Logo />)).toMatchSnapshot();
})