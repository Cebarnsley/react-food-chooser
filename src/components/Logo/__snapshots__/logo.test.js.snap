// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render logo component 1`] = `ShallowWrapper {}`;
